src/renderer/styles/global.css

```
@@ -66,6 +66,22 @@
 	--input: #e5e5e5;
 	--ring: #619eff;
 	--radius: 0.5rem;
+	--app-font-family-system:
+		"Monospaced Number", "Chinese Quote", -apple-system, BlinkMacSystemFont,
```

Stylelint (17.12.0)

[error] Expected "BlinkMacSystemFont" to be "blinkmacsystemfont" (value-keyword-case)

(value-keyword-case)